ngraph/core/CMakeLists.txt

```diff
@@ -33,8 +33,13 @@ source_group("include" FILES ${PUBLIC_HEADERS})
 
 configure_file(include/ngraph/version.in.hpp include/ngraph/version.hpp)
 
+if (NGRAPH_STATIC_LIBRARY)
+# Create static library
```

Comment:
nit: can we also add indent for comments?

Reply:
Checked